iothub/service/src/Utilities/NewtonsoftJsonETagConverter.cs

```diff
@@ -44,7 +44,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
             }
             else
             {
-                writer.WriteValue(eTag.ToString("H"));
+                writer.WriteValue(eTag.ToString());
```

Comment:
The json etag converter is used for serializing the object to be used in the request body. The "H" format tag is not valid for this scenario. The default tag is "G" -> standard formatting.

Reply:
Should we be explicit?